pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -2,25 +2,47 @@
 , csmSupport ? false, seabios ? null
 , secureBoot ? false
 , httpSupport ? false
+, crossIa32 ? false
```

Comment:
```suggestion
, crossIa32 ? (stdenv.isi686 || stdenv.isx86_64)
```

Not sure if this is correct. Maybe I missed a not.

Reply:
We do not want crossIa32 unless it's requested. We could technically configure two OVMF attributes under nixpkgs, or separate output for 32 and 64 bit.

Reply:
Thanks. Responded inline.